metadata-ingestion/src/datahub/cli/docker_check.py

```diff
@@ -189,28 +157,32 @@ def to_exception(
 
 def check_docker_quickstart() -> QuickstartStatus:
     container_statuses: List[DockerContainerStatus] = []
-
     with get_docker_client() as client:
         containers = client.containers.list(
             all=True,
             filters=DATAHUB_COMPOSE_PROJECT_FILTER,
         )
+        if len(containers) == 0:
+            return QuickstartStatus([])
+
+        # load the expected containers from the docker-compose file
+        config_file = containers[0].labels.get(
+            "com.docker.compose.project.config_files"
```

Comment:
what happens if they use multiple docker compose files

e.g. `docker-compose -p datahub -f compose1.yml -f compose2.yml`?

Reply:
address this.